packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -82,12 +82,34 @@ export interface EngineReportingOptions<TContext> {
    */
   reportErrorFunction?: (err: Error) => void;
   /**
-   * A case-sensitive list of names of variables whose values should not be sent
-   * to Apollo servers, or 'true' to leave out all variables. In the former
+   * [TO-BE-DEPRECATED] A case-sensitive list of names of variables whose values should
+   * not be sent to Apollo servers, or 'true' to leave out all variables. In the former
    * case, the report will indicate that each private variable was redacted; in
    * the latter case, no variables are sent at all.
    */
   privateVariables?: Array<String> | boolean;
+  /**
+   * A case-sensitive list of names of variables whose values should
```

Comment:
Does this stuff get extracted to user-visible docs (or shown in VS or something)? I don't know how I feel about the maintenance requirements of having full documentation of APIs in code that's then copy-and-pasted to md files. Though that seems to be the existing pattern...

Reply:
I'm not too sure if there's any syncing the docs, but I just copy-pasted here because it seemed to be the convention :\

Reply:
OK, no need to buck convention today.